src/content/drafts/redesigning-eslint.md

```diff
@@ -91,7 +91,11 @@ I also published the little Carbon Ads component as a [Figma Community file](htt
 
 ## The Playground
 
-The last part of the ESLint brand and redesign project was a new microsite called Playground! It's a fantastic little tool that allows you to create an ESLint configuration file with easy-to-use settings and test your code against it. Then, once you're happy with it, you can download your config file to use locally. 
+The last part of the project was a new microsite called Playground! It's a fantastic little tool that allows you to create an ESLint configuration file with easy-to-use settings and test your code against it. Then, once you're happy with it, you can download your config file to use locally. Its also a fantastic way of allowing new users to try ESLint directly in the browser.
+
+We started designing Playground with a few key requirements in mind — specifically we wanted to make it easy to use, and to make it easy to create and share reproducible bugs with ESLint. Users should also be able to switch between different versions of ESLint, see potential fixes to given problems and install plugins.
```

Comment:
Probably worth calling this out. 

```suggestion
We started designing Playground with a few key requirements in mind — specifically we wanted to make it easy to use, and to make it easy to create and share reproducible bugs with ESLint. Users should also be able to switch between different versions of ESLint, see potential fixes to given problems and install plugins. Ultimately the team couldn’t implement all of these features, but we designed for them all the same. 
```

Reply:
Ah fair! I thought they might be on the roadmap but all good, happy to add this.